metaflow/client/core.py

```diff
@@ -1280,23 +1323,28 @@ def environment_info(self):
         if not env_type:
             return None
         env = [m for m in ENVIRONMENTS + [MetaflowEnvironment] if m.TYPE == env_type][0]
-        return env.get_client_info(self.path_components[0], self.metadata_dict)
+        meta_dict = self.metadata_dict
+        return env.get_client_info(self.path_components[0], meta_dict)
```

Comment:
How is this change materially different from what existed before?

Reply:
Right, it isn't. Sorry, a bit too trigger happy in my changes. I'll revert it back.